src/Models/Esp32Device.ts

```diff
@@ -62,6 +63,8 @@ export class Esp32Device extends ArduinoDeviceBase {
 
   name = "Esp32Arduino";
 
+  private azureConfigFileHandler: AzureConfigFileHandler;

```

Comment:
Consider create AzureComponent class and move this field to the class since this field appears for many comnents whch needs azure configuration.

Reply:
Are you suggesting to define a static class to handle AzureConfig info? I've ever considered it. It will bring changes to lots of files. AzureConfigFileHandler class is an existing class for handling azure config file. Define a "azureConfigFileHandler" member in device/module classes is the current practice.